css/selectors/-webkit-scrollbar.json

```diff
@@ -36,7 +36,8 @@
               "version_added": "4"
             },
             "safari_ios": {
-              "version_added": "3"
+              "version_added": "3",
+              "version_removed": "14"
```

Comment:
It would be interesting to see if setting display: none on this property still works? I suspect it probably does?

Reply:
I tested this in BrowserStack. `display: none` does work. We should probably not apply this change.

Reply:
Thx for testing it @ddbeck i have not had the chance yet.